lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -716,11 +868,6 @@ def set_up_database(self) -> None:
                             Identifier(user)
                         )
                     )
-                self.create_user(
-                    PERMISSIONS_GROUP_ADMIN,
-                    extra_user_roles=["pg_read_all_data", "pg_write_all_data"],
-                )
```

Comment:
As this seems to be a breaking change, I think we need to bump the `LIBAPI` value to 1, so we can avoid needing to backport this change to PostgreSQL 14 or adapt this library to work on both PG versions.

Reply:
bumped postgresql lib to v1 in [539b82f](https://github.com/canonical/postgresql-operator/pull/837/commits/539b82f31c2f61c3a824a831f3fc224ea3c4ae53)

will open a separate PR shortly to add the file in the postgresql-k8s repo. this way, the lib can be cleanly published on charmhub

Reply:
Thanks!